File: CS/WindowsApplication3/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraWizard;

namespace WindowsApplication3 {
    public partial class Form1: Form {
        public Form1() {
            InitializeComponent();
        }
    }

    public class MyWizardControl: WizardControl {
        protected override WizardViewInfo CreateViewInfo() { 
            return new MyWizardViewInfo(this); 
        }
        
    }
    public class MyWizardViewInfo: WizardViewInfo {
        public MyWizardViewInfo(MyWizardControl w) : base(w) { }
        protected override WizardViewInfo.WizardModelBase CreateWizardModelCore(WizardStyle style) {
            if(style == WizardStyle.WizardAero) return new MyWizardAeroModel(this);
            return base.CreateWizardModelCore(style);
        }
    }
    public class MyWizardAeroModel: DevExpress.XtraWizard.WizardViewInfo.WizardAeroModel {
        public MyWizardAeroModel(MyWizardViewInfo v) : base(v) { }

        public override Rectangle GetContentBounds() {
            Rectangle rect = GetClientRectangle();
            rect.Y -= WizardAeroConsts.TitleBarHeight;
            rect.Height += WizardAeroConsts.TitleBarHeight;
            return rect;
        }

        public override Rectangle GetTitleBarBounds() {
            return Rectangle.Empty;
        }
    } 
}

File: VB/WindowsApplication3/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraWizard

Namespace WindowsApplication3
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
	End Class

	Public Class MyWizardControl
		Inherits WizardControl
		Protected Overrides Function CreateViewInfo() As WizardViewInfo
			Return New MyWizardViewInfo(Me)
		End Function

	End Class
	Public Class MyWizardViewInfo
		Inherits WizardViewInfo
		Public Sub New(ByVal w As MyWizardControl)
			MyBase.New(w)
		End Sub
		Protected Overrides Function CreateWizardModelCore(ByVal style As WizardStyle) As WizardViewInfo.WizardModelBase
            If style = WizardStyle.WizardAero Then
                Return New MyWizardAeroModel(Me)
            End If
			Return MyBase.CreateWizardModelCore(style)
		End Function
	End Class
	Public Class MyWizardAeroModel
		Inherits DevExpress.XtraWizard.WizardViewInfo.WizardAeroModel
		Public Sub New(ByVal v As MyWizardViewInfo)
			MyBase.New(v)
		End Sub

		Public Overrides Function GetContentBounds() As Rectangle
			Dim rect As Rectangle = GetClientRectangle()
			rect.Y -= WizardAeroConsts.TitleBarHeight
			rect.Height += WizardAeroConsts.TitleBarHeight
			Return rect
		End Function

		Public Overrides Function GetTitleBarBounds() As Rectangle
			Return Rectangle.Empty
		End Function
	End Class
End Namespace
